### PR TITLE
Fix cache support for some versions

### PR DESCRIPTION
**Pull request by alexbakharew**
### Scope & Purpose

Now we will take into account the fact, that primaryKeyCache and primarySortCache have bugs: https://arangodb.atlassian.net/browse/SEARCH-466

- [x] :hankey: Bugfix
- [x] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [x] sidelining PRs
  - [ ] [ArangoDB Pullrequest](https://github.com/arangodb/arangodb/pull/)
  - [x] [RTA Pullrequest](https://github.com/arangodb/release-test-automation/pull/353)

### CODE REVIEW

**Review by dothebart — CHANGES_REQUESTED** [review]
please fix the two version checks

File: test_data/makedata_suites/402_views.js [hunk 1]
```diff
@@ -722,12 +722,23 @@ function deleteAnalyzer_400(testgroup, analyzerName){
...
 
-    // remove redundant 'utilizeCache' values. 
+    // remove redundant 'utilizeCache' values.
     delete expected["utilizeCache"];
     if (expected.hasOwnProperty("collectionName")) {
       delete expected["collectionName"];
     }
 
+    let version = db._version();
+    if (semver.lt(version, "3.9.11") || semver.lt(version, "3.10.6")) {
```
[the remark this answers is not in the document]
Comment (alexbakharew) [reply test_data/makedata_suites/402_views.js]: Resolved

**Comment by alexbakharew** [comment]
Fix for https://arangodb.atlassian.net/browse/BTS-1352 will be introduced later

**Review by dothebart — APPROVED** [review]

**Comment by dothebart** [comment]
great, tnx!

**Pull request merged**